app/_hub/kong-inc/rate-limiting-advanced/_index.md

```diff
@@ -7,11 +7,12 @@ description: |
   The Rate Limiting Advanced plugin for Konnect Enterprise is a re-engineered version of the Kong Gateway (OSS) [Rate Limiting plugin](/hub/kong-inc/rate-limiting/).
 
   As compared to the standard Rate Limiting plugin, Rate Limiting Advanced provides:
-  * Additional configurations: `limit`, `window_size`, and `sync_rate`
+  * Enhanced capabilities to tune the rate limiter, provided by the parameters `limit` and `window_size`. Learn more in [Multiple Limits and Window Sizes](#multi-limits-windows)
   * Support for Redis Sentinel, Redis cluster, and Redis SSL
-  * Increased performance: Rate Limiting Advanced has better throughput performance with better accuracy. Configure `sync_rate` to periodically sync with backend storage.
+  * Increased performance: Rate Limiting Advanced has better throughput performance with better accuracy. The plugin allows tuning of performance and accuracy via a configurable synchronization of counter data with the backend storage. This can be controlled by setting the desired value on the `sync_rate` parameter.
```

Comment:
```suggestion
  * Increased performance: Rate Limiting Advanced has better throughput performance with better accuracy. The plugin allows you to tune performance and accuracy via a configurable synchronization of counter data with the backend storage. This can be controlled by setting the desired value on the `sync_rate` parameter.
```